components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtilTest.java

```diff
@@ -310,6 +317,8 @@ public void testGetUserConsentURL(Object oAuth2ParamObject, boolean isOIDC, bool
             Assert.assertTrue(consentUrl.contains(URLEncoder.encode("TestApplication", "ISO-8859-1")),
                     "application parameter value is not found in url");
             Assert.assertTrue(consentUrl.contains("scope2"), "scope parameter value is not found in url");
+            Assert.assertTrue(consentUrl.contains("internal_login"), "internal_login scope parameter " +
```

Comment:
Is this test valid? I see that the `JDBCPermissionBasedInternalScopeValidator ` is mocked and in the test, it is written to always send the `internal_login` login scope? so is there any scenario that this assertion can fail?

Reply:
With https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1766/commits/b4fdffd2d034206b71f43fb37879075ab7cd1d0b changes, Now we can test the flow of the newly added `validateScope` method.